spyder/plugins/editor/widgets/codeeditor/codeeditor.py

```diff
@@ -4522,43 +4565,46 @@ def mousePressEvent(self, event: QKeyEvent):
         """Override Qt method."""
         self.hide_tooltip()
 
-        ctrl = event.modifiers() & Qt.KeyboardModifier.ControlModifier
-        alt = event.modifiers() & Qt.KeyboardModifier.AltModifier
-        shift = event.modifiers() & Qt.KeyboardModifier.ShiftModifier
-        cursor_for_pos = self.cursorForPosition(event.pos())
-        self._mouse_left_button_pressed = event.button() == Qt.LeftButton
+        modifiers = event.modifiers()
+        left_button = event.button() == Qt.LeftButton
+        self._mouse_left_button_pressed = left_button
 
+        # Handle adding cursors
         if (
-            self.multi_cursor_enabled
-            and event.button() == Qt.LeftButton
-            and ctrl
-            and alt
+                self.multi_cursor_enabled and left_button and
+                modifiers == self.mouse_shortcuts['add_remove_cursor']
```

Comment:
```suggestion
            self.multi_cursor_enabled and left_button and
            modifiers == self.mouse_shortcuts['add_remove_cursor']
```

Fix Black formatting.